lib/tasks/multicoops.rake

```diff
@@ -8,8 +8,12 @@ namespace :multicoops do
   task :run => :environment do
     task_to_run = ENV['TASK']
     FoodsoftConfig.each_coop do |coop|
-      rake_say "Run '#{task_to_run}' for #{coop}"
-      Rake::Task[task_to_run].execute
+      begin
+        rake_say "Run '#{task_to_run}' for #{coop}"
+        Rake::Task[task_to_run].execute
+      rescue => error
+        ExceptionNotifier.notify_exception(error, data: {foodcoop: coop})
+      end
```

Comment:
Do we need to re-raise the error, so you see something is amiss when no exception notifier is configured?

Reply:
i see no need for that, since `multicoop` is usually only used in `production` environment, where it should be configured

Reply:
Some people may run without the notifier (for various reasons). Let's make it obvious when an error occurs, otherwise debugging becomes really hard. (Or maybe the notifier isn't configured properly.)

Reply:
done

Reply:
great, but why not re-raise it in the `rescue` block? like [here](https://www.exceptionalcreatures.com/guides/advanced-rescue-and-raise.html#reraising-exceptions)

Reply:
because then the first error will stop the whole process.
e.g. if banking import of one foodcooop fails all other will fail too, which is a problem

Reply:
super, I missed that before :)